src/client/pages/ConsentsCommunication.stories.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+
+import { ConsentsCommunication } from './ConsentsCommunication';
+
+export default {
+  title: 'Pages/ConsentsCommunication',
+  component: ConsentsCommunication,
+} as Meta;
+
+export const NoConsent = () => (
+  <ConsentsCommunication isUserInTest={false} consentsWithoutOptout={[]} />
+);
+NoConsent.story = {
+  name: 'with no consents',
+};
+
+export const MarketResearch = () => (
+  <ConsentsCommunication
+    isUserInTest={false}
+    marketResearchOptout={{
+      id: '0',
+      name: 'mr-name',
+      description:
+        'I do NOT wish to be contacted by The Guardian for market research purposes.',
+    }}
+    consentsWithoutOptout={[]}
+  />
+);
+MarketResearch.story = {
+  name: 'with market research consent',
+};
+
+export const WithoutOptout = () => (
+  <ConsentsCommunication
+    isUserInTest={false}
+    consentsWithoutOptout={[
+      {
+        id: '0',
+        name: 'My Consent Name',
+        description: 'Consent description',
+      },
+    ]}
+  />
+);
+WithoutOptout.story = {
+  name: 'with consents',
+};
+
+export const MultipleConsents = () => (
+  <ConsentsCommunication
+    isUserInTest={false}
+    consentsWithoutOptout={[
+      {
+        id: '0',
+        name: 'My Consent Name',
+        description: 'Consent description',
+      },
+      {
+        id: '1',
+        name: 'My Other Consent Name',
+        description: 'Other consent description',
+      },
+    ]}
+  />
+);
+MultipleConsents.story = {
+  name: 'with multiple consents',
+};
+
+export const MultipleConsentsAndMarketing = () => (
+  <ConsentsCommunication
+    isUserInTest={false}
+    marketResearchOptout={{
+      id: '0',
+      name: 'mr-name',
+      description:
+        'I do NOT wish to be contacted by The Guardian for market research purposes.',
+    }}
```

Comment:
this consent is not showing up in the story

Reply:
when the user is not in test `isUserInTest = false`, this consent should show

Reply:
ah interesting, it's showing up now, i have a feeling that it was due to the name clashing with the other story before